element_array_ephys/ephys.py

```diff
@@ -442,31 +449,34 @@ class Curation(dj.Manual):
     curation_id: int
     ---
     curation_time: datetime             # time of generation of this set of curated clustering results 
-    curation_output_dir: varchar(255)   # output directory of the curated results, relative to clustering root data directory
+    curation_output_dir: varchar(255)   # output directory of the curated results, relative to root data directory
     quality_control: bool               # has this clustering result undergone quality control?
     manual_curation: bool               # has manual curation been performed on this clustering result?
     curation_note='': varchar(2000)  
     """
 
     def create1_from_clustering_task(self, key, curation_note=''):
         """
-        A convenient function to create a new corresponding "Curation"
-         for a particular "ClusteringTask"
+        A function to create a new corresponding "Curation" for a particular 
+        "ClusteringTask", which assumes that no curation was performed on the 
+        dataset
```

Comment:
I don't think we should explicitly say `which assumes that no curation was performed on the dataset` here. 
Technically users can still use this convenient function to create a new entry to this table using info from the Clustering Task table, for a ***new*** curation that they've done (e.g. they just put the new curated outputs in the same previous directory, overwriting the old results)

Reply:
Thanks for clarifying.  I removed this phrase.